lib/rubocop/cop/style/comment_annotation.rb

```diff
@@ -40,6 +40,13 @@ module Style
       #   # good
       #   # OPTIMIZE: does not work
       #
+      #   # bad
+      #   # HACK:
+      #   # does not work
+      #
+      #   # good
+      #   # HACK: does not work
```

Comment:
Hm, user may have intentionally wrote this, so this cop may need to respect user intent. There is also the possibility of avoiding `Layout/LineLength` offense for example. WDYT?

Reply:
I mentioned in another comment https://github.com/rubocop/rubocop/pull/10836#issuecomment-1195043385 , it will always be offense and not autocorrection.

The following files
```ruby
# frozen_string_literal: true

# NOTE:
# This is the text of the note
# :
```

Run rubocop -A
```command
bundle exec rubocop -A test.rb               
Inspecting 1 file
C

Offenses:

test.rb:3:3: C: Style/CommentAnnotation: Annotation comment, with keyword NOTE, is missing a note.
# NOTE:
  ^^^^^

1 file inspected, 1 offense detected
```

But, they are not autocorrected.

Reply:
I found it very painful when there are many todo's to `.rubocop_todo.yml` and I wanted to deal with them all at once.
For example, I thought it might be a good idea to add another option that would toggle this behavior. WDYT?

Reply:
I was mistaken. As you pointed out, it is no good to always autocorrect, as it is an offense by `Layout/LineLength` in some cases. Thank you very much.